lib/charms/traefik_k8s/v2/ingress.py

```diff
@@ -13,7 +13,7 @@
 
 ```shell
 cd some-charm
-charmcraft fetch-lib charms.traefik_k8s.v2.ingress
+charmcraft fetch-lib charms.traefik_k8s.v1.ingress
```

Comment:
...this is an odd regression?

Reply:
Yea I noticed too. Should we open a pr or notify someone?